### PR TITLE
Update devices-wipe.md

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the navigation to Retire an Apple ADE device from Intune

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3915

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 1]
```diff
@@ -223,9 +223,9 @@ If you want to completely remove an Apple automated device enrollment (ADE) devi
 1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 2. Choose **Devices** > **All devices** > choose the device > **Retire**.
 3. Visit [business.apple.com](http://business.apple.com) and search for the device by its serial number.
-4. In the **Assigned to** menu, choose **Unassigned**.
+4. Select the Device , choose **Release from Organization**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/devices-wipe.md:226]:
```suggestion
4. Select the device and choose **Release from Organization**.
```

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 2]
```diff
@@ -223,9 +223,9 @@ If you want to completely remove an Apple automated device enrollment (ADE) devi
...
 
-5. Choose **Reassign**.
+5. Check **I understand this cannot be undone** and click on **Release**
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/devices-wipe.md:228]:
```suggestion
5. Check **I understand this cannot be undone** and select **Release**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: memdocs/intune/remote-actions/devices-wipe.md (hunk 1 above)
Comment (tiburd) [inline memdocs/intune/remote-actions/devices-wipe.md:226]:
```suggestion
4. Select the device, and then select **Release from Organization**.
```

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 3]
```diff
@@ -223,9 +223,9 @@ If you want to completely remove an Apple automated device enrollment (ADE) devi
 1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 2. Choose **Devices** > **All devices** > choose the device > **Retire**.
 3. Visit [business.apple.com](http://business.apple.com) and search for the device by its serial number.
-4. In the **Assigned to** menu, choose **Unassigned**.
+4. Select the device, and then select **Release from Organization**.
 
-5. Choose **Reassign**.
+5. Check **I understand this cannot be undone** and click on **Release**
```
Comment (tiburd) [inline memdocs/intune/remote-actions/devices-wipe.md:228]:
```suggestion
5. Check **I understand this cannot be undone**, and then select **Release**.
```

**Pull request merged**